app/controllers/monitor_controller.rb

```diff
@@ -1,6 +1,6 @@
 class MonitorController < ApplicationController
   def create
-    monitor = LinkMonitor::CreateResourceMonitor.new(links: permitted_params[:links], service: permitted_params[:service]).call
+    monitor = LinkMonitor::UpsertResourceMonitor.new(links: permitted_params[:links], service: permitted_params[:service], resource_type: permitted_params[:resource_type], resource_id: permitted_params[:resource_id]).call
```

Comment:
Might be worth making `permitted_params` return using `.to_h` and then can just do `UpsertResourceMonitor.new(**permitted_params)` here.